packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts

```diff
@@ -217,6 +218,15 @@ MeetingInfoUtil.getDestinationType = async (from) => {
   return Promise.resolve(options);
 };
 
+/**
+ * Predicate function to be used when filtering request params.
+ * Currently this simply checks for truthy values but in future could restrict keys and values.
+ * @param {Any} value the request param value
+ * @param {String} key the request param key
+ * @returns {Boolean} if the key and value are valid as a request param
+ */
+MeetingInfoUtil.isValidRequestParam = (value: any, key: string): boolean => !!(key && value);
```

Comment:
question: I'm not sure what's the benefit of using this predicate in its current form. What if someone wants to add an extra param that has a false-y value? Can't we just spread all the extraParams without the call to pickBy?

Reply:
> My reasoning for adding the predicate was actually to eventually remove the explicit installedOrgID and locusId.... 

Sounds like a good idea, but realistically I don't think this will ever get done unless you do it now in this PR, so I would either do it fully now or remove the predicate.

My 2nd concern is that sometimes we actually need false-y values to be sent to server (like empty string), so that's why I feel we shouldn't be filtering out them, we can filter out just false-y keys

Reply:
As discussed, I have removed the predicate.

https://github.com/webex/webex-js-sdk/pull/2807/commits/0bf745d36489c2b395d6fa1aeb97166be27cdd9d